joomla/joomla-cms
# [5.0] Smart Search: Fixing taxonomy filter handling

Pull Request for Issue #39718 .

### Summary of Changes
When I refactored the taxonomies to be a nested set, I overlooked this part. This means that nested taxonomies in filters don't work.


### Testing Instructions
1. Create an article in a category which is at least on the second level.
2. Create a filter which only searches in that articles parent category. 
3. Go to the frontend and search for a word in that article. You will get the article as result.
4. Add &f=[filter-id] to the URL to only show results from our category.
5. Notice that you get no result at all!
6. Apply changes.
7. Only the article appears again.

### Link to documentations
Please select:


@crystalenka can you check if this fixes your issue?
